src/utils/command.rs

```diff
@@ -274,7 +283,88 @@ mod tests {
 
         assert_eq!(
             DefaultCommands::Start,
-            DefaultCommands::parse("/start@MyNameBot", "MyNameBot").unwrap().0
+            DefaultCommands::parse("/start@MyNameBot", "MyNameBot").unwrap()
+        );
+    }
+
+    #[test]
+    fn parse_with_split() {
+        #[command(rename = "lowercase")]
```

Comment:
`#[command]` before `#[derive]` is a bit confusing

Reply:
Fixed.